lib/Common/Memory/Recycler.h

```diff
@@ -1737,7 +1785,10 @@ class Recycler
 
     template <CollectionFlags flags>
     BOOL TryFinishConcurrentCollect();
-    BOOL WaitForConcurrentThread(DWORD waitTime);
+
+    BOOL WaitForConcurrentThread(DWORD waitTime, WaitForConcurrentCaller caller = OTHER);
```

Comment:
Can we fix the casing and use an enum class for WaitForConcurrentCaller? The current casing makes it look like a macro...

Reply:
fixed